repos/core-setup.proj

```diff
@@ -34,6 +34,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <RepositoryReference Include="aspnet-extensions" />
```

Comment:
aspnet-extensions comes after core-setup. The dependency flow of repos is:

```
dotnet/core-setup
 -> aspnet/Extensions
    -> aspnet/AspNetCore-Tooling 
        -> dotnet/toolset
        -> aspnet/AspNetCore
    -> aspnet/AspNetCore 
        -> aspnet/websdk
        -> dotnet/core-sdk
```

Reply:
Fixed the repo dependencies.